torch/onnx/symbolic_opset9.py

```diff
@@ -267,6 +267,8 @@
     "unsafe_split_with_sizes",
     "unsafe_split",
     "unsqueeze",
+    "_unsupported_complex_operators",
+    "_unsupported_complex_operators_noop",
```

Comment:
```suggestion
```

no need to expose private functions. This will error in CI anyways.

Reply:
Changed to public again. Any symbolic that might be decomposed in other symbolics to manipulate complex can reuse it. How we handle no-op can change and reusing prevents duplicates and inconsistencies

Reply:
In my understanding the issues were resolved, and therefore comment was closed

1) They were changed to public as they can be reused can be reused by other symbolics that decompose in more than ONNX operator, possibly having to parse complex types. 
2) A symbolic with noop and another with raise were also created as suggested.

Reply:
I think conversations like these where decisions are made during the review are helpful to be left open so future readers and reviewers can refer back to them.

An open thread doesn't block the pr or mean an unresolved issue and I always find them helpful for understanding contexts. I usually allow reviewers to close comments to make sure no follow ups are needed.